### ISSUE TITLE
plot_model and contrast options

### ISSUE THREAD

**Issue by nfaivre** [strengejacke/sjPlot#442]
Hi there. First, thanks for this excellent package!

I noticed a strange behavior, which I'm not sure is normal. It seems like confidence intervals from a binomial mixed model change depending on the contrast option. 

For instance, if I run:
`options(contrasts = rep("contr.treatment", 2))`
`m = glmer(cor ~ conf * cond  + (conf*cond| suj) , data = a, family = binomial)`

then `plot_model(m,type = "pred",terms=c('conf [all]','cond')` gives me a nice sigmoid, as expected. 
If I run the exact same model, preceded by `options(contrasts = rep("contr.sum", 2))` I get the same sigmoid, but with confidence intervals around it stopping at x =0:
![screenshot from 2018-11-28 10-51-09](https://user-images.githubusercontent.com/34683012/49143546-8c569e00-f2fb-11e8-91bf-1cad214763bd.png)

Any idea where this comes from? And any way we can get full CI with sum contrasts? I can try to provide a reproducible example if it's useful, but the data I used for this case is too big to share here.

**Comment by strengejacke** [comment]
Yes, please provide a small, reproducible example (with reduced dataset if possible and necessary), so I can dig further into this. I think this might be because axis limits are too narrow, but I must check this with a working example.

**Comment by nfaivre** [comment]
Here is the reproducible example. What is strange is that confidence intervals change depending on the factor labels (here, a$cond). 

## load data
[a_test.zip](https://github.com/strengejacke/sjPlot/files/2625652/a_test.zip)

rm(list=ls(all=TRUE))
library(lme4)
library(sjPlot)
load('./a_test.RData')
glimpse(a)

## treatment contrast, ok plot
options(contrasts = rep("contr.treatment", 2))
m=glmer(cor ~ conf * cond + (conf* cond| suj) , data = a, family = binomial) 
plot_model(m,type = "pred",terms=c('conf [all]','cond'))

## sum contrast, increase in CI 
options(contrasts = rep("contr.sum", 2))
m1=glmer(cor ~ conf* cond  + (conf* cond| suj) , data = a, family = binomial) 
plot_model(m1,type = "pred",terms=c('conf [all]','cond'))


## change cond labels, back to ok plot even with sum contrast
a$cond=ifelse(a$cond=='active','1','0')
a$cond=as.factor(a$cond)
m2=glmer(cor ~ conf* cond  + (conf* cond| suj) , data = a, family = binomial) 
plot_model(m2,type = "pred",terms=c('conf [all]','cond'))

**Comment by strengejacke** [comment]
setting `active` to `1` and `observation` to `0` changes the reference category and leads to different estimates in the model. That might be a reason why `m1` does not work, but `m2` does.

I now need to figure out why `m1` produces so many missing values for the CIs.

**Comment by strengejacke** [comment]
I think I fixed the bug. You need to install **ggeffects** from GitHub to check. The problem was that `as.data.frame(model.matrix(model))` behaves differently in naming columns depending on the contrasts-option you set.

I'm not sure if I now covered all exceptions, but at least your issue should be solved.

**Comment by nfaivre** [comment]
Great, thanks for looking into it so quickly!
